roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml

```diff
@@ -0,0 +1,227 @@
+---
+- name: generate hawkular-metrics certificates
+  include: setup_certificate.yaml
+  vars:
+    component: hawkular-metrics
+    hostnames: "hawkular-metrics,{{ openshift_metrics_hawkular_metrics_hostname }}"
+- name: generate hawkular-cassandra certificates
+  include: setup_certificate.yaml
+  vars:
+    component: hawkular-cassandra
+    hostnames: hawkular-cassandra
+- name: check existing aliases on the hawkular-cassandra truststore
+  shell: >
+    keytool -noprompt -list
```

Comment:
This is good for now but is the same issue we have for logging in that we need to move to not require a user to have java installed.  That was a concern from the ansible team.  This is work that eric is doing with the java pod or the suggestion jimmy dyson had to convert the pem to jks using an init container